compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs

```diff
@@ -864,14 +868,9 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
             } else {
                 bug!("not an upvar")
             };
-            err.span_label(
-                *span,
-                format!(
-                    "calling `{}` requires mutable binding due to {}",
-                    self.describe_place(the_place_err).unwrap(),
-                    reason
-                ),
-            );
+            let place = self.describe_place(the_place_err).unwrap();
```

Comment:
We'll need to be sure that `describe_place` doesn't include English words that won't be translatable.

Reply:
The `reason` is where filled with English words. I hope that explains.

I think I'll deal with the nested cases after I finish the trivial ones.